modules/v2x/fusion/libs/common/base/v2x_object_types.h

```diff
@@ -0,0 +1,214 @@
+/******************************************************************************
```

Comment:
It seems that many of classes defined in this file are not used, and there are many duplicates in modules/perception/base/object_types.h. So do some cleanup.